.github/workflows/src-mirror.yml

```diff
@@ -13,10 +13,17 @@ jobs:
         with:
           git-fetch-depth: 0
           path: workspace/nrf
+          west-update-args: '--narrow -o=--depth=1 --group-filter=+babblesim,+sidewalk,+optional'
```

Comment:
+optional is default so it can be dropped.

Reply:
I think we should remove `--narrow -o=--depth=1` 

users might need other refs and history.